tarantool/src/main/java/com/yahoo/ycsb/db/TarantoolClient.java

```diff
@@ -19,28 +19,25 @@
 import com.yahoo.ycsb.*;
 import org.tarantool.TarantoolConnection16;
 import org.tarantool.TarantoolConnection16Impl;
-import org.tarantool.TarantoolException;
```

Comment:
This is done for the previous review iteration of review. I can revert back to previous state or state currently in master or whatever you like, just tell what is the preferred way (preferably it should be preferable to all reviews or at least till the end if this particular review)

Reply:
exceptions that come out of the DB layer will eventually cause the Client to die. In general, we expect report errors from underlying storage rather than abort the test run. Thus, the preferred approach is for bindings to do error handling for the datastore they are talking with.